pkgs/applications/graphics/geeqie/default.nix

```diff
@@ -56,7 +83,7 @@ stdenv.mkDerivation rec {
 
     license = licenses.gpl2Plus;
 
-    homepage = "http://geeqie.sourceforge.net";
+    homepage = "https://github.com/BestImageViewer/geeqie";
 
     maintainers = with maintainers; [ jfrankenau pSub markus1189 ];
```

Comment:
Can you remove the extra new lines from meta?

Reply:
Yes, done.